src/test/java/core/PluginManagerTest.java

```diff
@@ -52,9 +53,9 @@ public void uninstall_plugin() throws InterruptedException, ExecutionException {
         check(find(by.url("plugin/gerrit-trigger")), false);
         WebElement form = find(by.action("plugin/gerrit-trigger/uninstall"));
         form.submit();
+        clickButton("Yes");
```

Comment:
well apparently this test didn't actually uninstall plugins =/ it didn't confirm the uninstall 🤦